seml/settings.py

```diff
@@ -1,14 +1,14 @@
 from munch import munchify
-from pathlib import Path
-
+import os
+import seml
 __all__ = ("SETTINGS",)
-
+seml_base = os.path.dirname(os.path.abspath(seml.__file__))
 
 SETTINGS = munchify(
     {
         "DATABASE": {
-            # location of the MongoDB config. Default: $HOME/.config/seml/monogdb.config
-            "MONGODB_CONFIG_PATH": f'{str(Path.home())}/.config/seml/mongodb.config'
+            # location of the MongoDB config. Default: /path/to/seml/monogdb.config
+            "MONGODB_CONFIG_PATH": f'{seml_base}/mongodb.config'
```

Comment:
Why are we moving the config back into the seml directory?

Reply:
I thought it's good to have everything in one place. Do you prefer the old location?

Reply:
I do, yes. It's the same discussion as last time. User-specific settings are usually saved in `~/.config`. For the user, the seml path will look like `~/anaconda/envs/pytorch/lib/python3/site-packages/seml`. Not quite intuitive or discoverable. ;)

Reply:
Makes sense. 02abc9c reverts this change.